oauth2_http/java/com/google/auth/oauth2/StsRequestHandler.java

```diff
@@ -175,6 +193,11 @@ private StsTokenExchangeResponse buildResponse(GenericData responseData) throws
       String scope = OAuth2Utils.validateString(responseData, "scope", PARSE_ERROR_PREFIX);
       builder.setScopes(Arrays.asList(scope.trim().split("\\s+")));
     }
+    if (responseData.containsKey("access_boundary_session_key")) {
+      builder.setAccessBoundarySessionKey(
+          OAuth2Utils.validateString(
+              responseData, "access_boundary_session_key", PARSE_ERROR_PREFIX));
+    }
```

Comment:
nit: Perhaps `access_boundary_session_key` as a package-private constant and we can use this in the tests as well?

Reply:
Ah, the reason I was suggesting it is because I saw a test below had:
`private static final String ACCESS_BOUNDARY_SESSION_KEY = "accessBoundarySessionKey";` and assumed it was for the key (it looks like it's for the value instead).

nit: IMO, ACCESS_BOUNDARY_SESSION_KEY should reference `access_boundary_session_key`. If the constant probably be renamed to be something like ACCESS_BOUNDARY_SESSION_KEY_VALUE